app/services/actual_overview.rb

```diff
@@ -1,7 +1,8 @@
 class ActualOverview
-  def initialize(activity, report)
+  def initialize(activity:, report:, include_adjustments: false)
```

Comment:
I wonder if we should maybe push ourselves to be clear on where it's appropriate for an `ActualOverview` to **not** include adjustments? 

I see that `Actual#actual_total_for_report_financial_quarter` is used in the "variances" tab, but is it really more correct for adjustments to be excluded from variances? i.e. I think need to see if we can simplify the business rules and the code by making the inclusion of adjustments a new element, rather than an _optional_ new element...

If what I suggest **does** make sense, probably that would be a clarification to make in a separate PR?

Reply:
I think the reason I did it this way is because this class gets used elsewhere, and I wanted to make sure we did the simplest thing possible in the first instance. We have a card elsewhere that covers including adjustments in calculations, and I wanted to make sure we didn't touch other parts of the codebase and complicating the feature. We can (and should) tidy this up later.